src/m365/connection/commands/connection-use.ts

```diff
@@ -25,18 +27,41 @@ class ConnectionUseCommand extends Command {
     super();
 
     this.#initOptions();
+    this.#initTelemetry();
+    this.#initTypes();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        name: typeof args.options.name !== 'undefined'
+      });
+    });
   }
 
   #initOptions(): void {
     this.options.unshift(
       {
-        option: '-n, --name <name>'
+        option: '-n, --name [name]'
       }
     );
   }
 
+  #initTypes(): void {
+    this.types.string.push('name');
+  }
+
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    const connection = await auth.getConnection(args.options.name);
+    let connection: Connection;
+    if (args.options.name) {
+      connection = await auth.getConnection(args.options.name);
+    }
+    else {
+      let connections = await auth.getAllConnections();
+      connections = connections.sort((a, b) => a.name!.localeCompare(b.name!));
```

Comment:
The sort method sorts the array in place and returns the sorted array. So no need to reassign it.

```suggestion
      const connections = await auth.getAllConnections();
      connections.sort((a, b) => a.name!.localeCompare(b.name!));
```